profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,12 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-05-08)
+# Having a masked gentoo-vm flag allows us to unmask virtual/{jdk,jre}:17
+# without breaking Gentoo packages.
```

Comment:
this is user-visible message, not a code comment.
please change to something like
`gentoo-vm useflag is not stable yet`

Reply:
Done.